api/src/test/java/org/openmrs/notification/MessageServiceTest.java

```diff
@@ -114,4 +116,24 @@ public void sendMessage_shouldSendMessage() throws Exception {
 		}
 	}
 	
+	@Test
+	public void sendMessage_shouldSaveTheSentMessage() throws MessageException {
+		Message message = ms.createMessage("recipient@example.com", "sender@example.com", "subject", "content");
+		message.setId(123);
+		
+		try {
+			ms.sendMessage(message);
+			SentMessage sentMessage = Context.getSentMessageService().getSentMessage(123);
+			
+			assertNotNull(sentMessage);
+			assertEquals(123, (int)sentMessage.getMessageId());
+		}
+		catch (MessageException e) {
```

Comment:
I picked that convention from other tests in the file. Look at this [line](https://github.com/kishoreyekkanti/openmrs/blob/master/api/src/test/java/org/openmrs/notification/MessageServiceTest.java#L100)